pkg/backend/httpstate/client/client.go

```diff
@@ -1551,6 +1592,40 @@ func (pc *Client) PublishPackage(ctx context.Context, input apitype.PackagePubli
 	return nil
 }
 
+func (pc *Client) StartTemplatePublish(
+	ctx context.Context,
+	source, publisher, name string,
+	version semver.Version,
+) (*StartTemplatePublishResponse, error) {
+	req := StartTemplatePublishRequest{
+		Version: version,
+	}
+	var resp StartTemplatePublishResponse
+	err := pc.restCall(ctx, "POST", publishTemplatePath(source, publisher, name), nil, req, &resp)
+	if err != nil {
+		return nil, fmt.Errorf("start template publish failed: %w", err)
+	}
+	return &resp, nil
+}
+
+func (pc *Client) CompleteTemplatePublish(
+	ctx context.Context,
+	source, publisher, name, version string,
```

Comment:
If `StartTemplatePublish` takes a `version semver.Version`, then `CompleteTemplatePublish` should too.
```suggestion
	source, publisher, name, version semver.Version,
```